cmake/kokkos_configure_trilinos.cmake

```diff
@@ -33,6 +33,8 @@ if(CMAKE_PROJECT_NAME STREQUAL "Trilinos")
 
   set(Kokkos_ENABLE_COMPLEX_ALIGN OFF CACHE BOOL "Whether to align Kokkos::complex to 2*alignof(RealType)")
 
+  set(Kokkos_ENABLE_IMPL_VIEW_LEGACY ON CACHE BOOL "Whether to use the legacy implementation of View" FORCE)
```

Comment:
I am not sure I would want us to do it here but rather Trilinos take care of this themselves...Just as they should be tracking when they can switch to using the mdspan impl

Reply:
As in you had initially added the FORCE keyword L34 as part of this PR but you decided to withdraw that change?

Do we actually need that FORCE trailing argument in the set L36 or did you add it in the name of consistency.